.github/CODEOWNERS

```diff
@@ -9,46 +9,46 @@
 /sdk/                                           @AlexGhiondea
 
 # Core
-/sdk/core/                                      @pakrym @KrzysztofCwalina
+/sdk/core/                                      @pakrym @KrzysztofCwalina       %Azure.Core
```

Comment:
Given we cannot easily align these tags I would just add them all to the end of the users but not a big deal.

Reply:
Makes sense. Done.